<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detailClustered</class>
 <widget class="QMainWindow" name="detailClustered">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>detailClustered</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeDetail">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>685</width>
      <height>461</height>
     </rect>
    </property>
    <property name="headerHidden">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>1</number>
    </property>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
